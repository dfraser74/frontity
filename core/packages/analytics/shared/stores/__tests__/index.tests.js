import { Stores } from '../mocks';

describe('Analytics', () => {
  test('view - customDimensions', () => {
    const stores = Stores.create();
    expect([
      stores.analytics.customDimensions({ type: 'post', id: 57 }),
      stores.analytics.customDimensions({ type: 'post', id: 54 }),
      stores.analytics.customDimensions({ type: 'post', id: 34 }),
    ]).toMatchSnapshot();
  });
});
