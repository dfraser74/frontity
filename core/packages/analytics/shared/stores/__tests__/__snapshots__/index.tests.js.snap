// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Analytics view - customDimensions 1`] = `
Array [
  Object {
    "dimension1": "dim1_post57",
    "dimension2": "dim2_post57",
  },
  Object {
    "dimension1": "dim1_post54",
    "dimension2": "dim2_post54",
  },
  Object {},
]
`;
